[docs] Actions Refactoring (#4844)
DevExpress/testcafe: docs/articles/documentation/reference/test-api/testcontroller/clearupload.md

---
layout: docs
title: t.clearUpload Method
permalink: /documentation/reference/test-api/testcontroller/clearupload.html
---
# t.clearUpload Method

Removes all file paths from the specified file upload input.

```text
t.clearUpload( selector )
```

Parameter  | Type                                              | Description
---------- | ------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------

> `selector` | Function &#124; String &#124; Selector &#124; Snapshot &#124; Promise | Identifies the input field that needs to be cleared. See [Select Target Elements](#select-target-elements).

Reviewer: ```suggestion
`selector` | Function &#124; String &#124; Selector &#124; Snapshot &#124; Promise | Identifies the input field that should be cleared. See [Select Target Elements](#select-target-elements).
```
